config/initializers/active_support_extensions.rb

```diff
@@ -0,0 +1,30 @@
+#this makes sure that hash with indifferent access gets deep converted into hash.
```

Comment:
nah, I'll just remove it once it's rebased into the rails 4 branch.

On Thu, Feb 12, 2015 at 4:04 PM, Ivan Necas <notifications@github.com>
wrote:

> In config/initializers/active_support_extensions.rb
> <https://github.com/theforeman/foreman/pull/2148#discussion_r24582170>:
>
> > @@ -0,0 +1,30 @@
> > +#this makes sure that hash with indifferent access gets deep converted into hash.
>
> What about putting here some condition on Rails.version < 4, maybe with
> some warning in else to make sure we remove this code once not needed. Also
> a comment about the origin of the code would be nice
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/2148/files#r24582170>.
>